vendor/terser@4.8.0/terser.ts

```diff
@@ -5,5 +5,6 @@ import { AST_Node, MinifyOptions, MinifyOutput } from "./terser.d.ts";
 
 export const minify: (
   files: string | string[] | { [file: string]: string } | AST_Node,
-  options?: MinifyOptions
+  options?: MinifyOptions,
+// deno-lint-ignore no-explicit-any
```

Comment:
I'm going to land the Terser PR first so you'll probably have to rebase/update this part for the 5.3 version, I think?